platform/security/servlet/security-servlet-logout/src/main/webapp/js/main.js

```diff
@@ -40,7 +40,8 @@
                     .click(function () {
                         $('iframe').attr('src', action.url);
                         $('#modal').removeClass('is-hidden');
-
+                        window.localStorage.removeItem('systemUsage');
```

Comment:
systemUsage is set if the user has seen the modal that they need to consent to. It needs to be removed at logout so that when Person 2 logs in after Person 1 logs out out, Person 1's consent isn't in the local storage. Clearing systemUsage at logout ensures that the modal will display at every login.